docs/internals/lsm.md

```diff
@@ -117,8 +117,9 @@ optimization should enable the tree's performance to approach that of an append-
 
 #### Compaction Table Overlap
 
-When compacting a table from level `A` to level `B`, what is the maximum number of level-`B` tables
-that may overlap the level-`A` table (i.e. the "worst case")?
+Applying [this](#compaction-selection-policy) selection policy while compacting a table
+from level A to level B, what is the maximum number of level-B tables that may overlap with the
+selected level-A table (i.e. the “worst case”)?”
```

Comment:
Extra `“` at the end. Also, can you use regular `"` instead of the stylized ones? The play nicer with text editors.

Reply:
Oops, stray quote. Ack, I'll change this use regular `"`!